src/overlays/actors/ovl_En_Takara_Man/z_en_takara_man.c

```diff
@@ -27,25 +34,201 @@ const ActorInit En_Takara_Man_InitVars = {
     (ActorFunc)EnTakaraMan_Update,
     (ActorFunc)EnTakaraMan_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Takara_Man/EnTakaraMan_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Takara_Man/EnTakaraMan_Init.s")
+static u8 sTakaraIsInitialized = false;
+
+extern SkeletonHeader D_06004FE0;
+extern AnimationHeader D_06000498;
+
+void EnTakaraMan_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EnTakaraMan_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnTakaraMan* this = THIS;
+
+    if (sTakaraIsInitialized) {
+        Actor_Kill(&this->actor);
+        osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ もういてる原 ☆☆☆☆☆ \n" VT_RST); // "Already initialized"
+        return;
+    }
+
+    sTakaraIsInitialized = true;
+    osSyncPrintf("\n\n");
+    osSyncPrintf(VT_FGCOL(PURPLE) "☆☆☆☆☆ ばぅん！ ☆☆☆☆☆ %x\n" VT_RST,
+                 globalCtx->actorCtx.flags.chest); // "Bun! %x" (needs a better translation)
```

Comment:
in cases where it makes a new line like this id leave the translation on top.
additionally imo if the translation makes no sense and doesnt add any new information, might as well leave it out and let a translator do it later